.rubocop.yml

```diff
@@ -543,6 +543,3 @@ Style/FormatStringToken:
 # are located.
 RSpec/FilePath:
   Enabled: false
-
-RSpec/MessageSpies:
```

Comment:
This is needed. I didn't see an easy way around this and perhaps we need something in our modulesync to allow for this